webpack/react_app/components/RegistrationExtension/RexPull.js

```diff
@@ -23,6 +25,25 @@ const options = (value = '') => {
   );
 };
 
+const pullWarning = (
+  <Alert type="info" isInline style={{ marginTop: '10px' }}>
+    {__(
+      'Please make sure that the Smart Proxy is configured correctly for the Pull provider.'
+    )}
+  </Alert>
+);
+
+function showPullWarning(valueFromParam, value) {
```

Comment:
Note: The `if` branching might look weird at first glance, but it's because of:

`valueFromParam` can be `true`, `false` or `undefined`
`value` can be `'true'`, `'false'`, `''` or `undefined`.

It was fun.